test/data/router-methods-helper.js

```diff
@@ -0,0 +1,9 @@
+self.getMethodToTest = () => {
+  const methodRegex = /\?method=(\w*)/;
+  const methodRegexResult = methodRegex.exec(location.search);
```

Comment:
yeah - realistically I don't expect this test to get to a point where it will need more than this so I'd be inclined to leave it with regex for now.